python_codes/requirements.txt

```diff
@@ -2,4 +2,5 @@ pandas
 tqdm
 string
 openpyxl
-numpy
\ No newline at end of file
+numpy
```

Comment:
This file is not used. I propose that you remove it, and add your requirements in the `requirements.txt` file ?